samples/interoperability/qrng/NuGet.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <packageSources>
+    <add key="qdk-alpha" value="https://pkgs.dev.azure.com/ms-quantum-public/Microsoft Quantum (public)/_packaging/alpha/nuget/v3/index.json" protocolVersion="3" />
+    <add key="local" value="/home/mathias/code/nugets" />
```

Comment:
It looks like this line may not have been intended for inclusion in the sample?

Reply:
In fact, the whole file is not intended for inclusion. Might have added it by mistake.